src/components/button/ButtonToggle.tsx

```diff
@@ -0,0 +1,24 @@
+import cc from "classcat";
+
+import { Button } from "@/components/button";
+
+type ButtonToggleProps = {
+  // クリック前のボタン内テキスト
+  initialText: string;
+  // クリック後のボタン内テキスト
+  clickedText: string;
+  // クリック済みかどうか
+  isChecked?: boolean;
+};
+
+export const ButtonToggle: React.FC<ButtonToggleProps> = (props) => {
+  return !props.isChecked ? (
+    <Button bgColor="tomato" fontColor="white" size="small">
+      {props.initialText}
+    </Button>
+  ) : (
+    <Button bgColor="white" fontColor="tomato" size="small" isBorder>
+      {props.clickedText}
+    </Button>
+  );
```

Comment:
下の書き方もできるのですが
実装者の好み感はあるので微妙だと思ったらスルーしていただいて問題ないです。
```suggestion
return (
    <Button bgColor={props.isChecked ? "white" : "tomato"} fontColor={props.isChecked ? "tomato" : "white"} size="small">
      {props.isChecked ? props.clickedText : props.initialText}
    </Button>
  );
```